posts/contributions-to-mathlib-from-lte.md

```diff
@@ -0,0 +1,50 @@
+---
+author: 'Riccardo Brasca'
+category: 'New in mathlib'
+date: 2021-10-19 11:04:21 UTC+02:00
+description: ''
+has_math: true
+link: ''
+slug: contributions-to-mathlib-from-lte
+tags: ''
+title: Contributions to mathlib from LTE
```

Comment:
The title should either be more specific (i.e. include "normed groups" in the text), or at least briefly refer to all the other contributions to mathlib from LTE!